cmd/capslock-git-diff/main.go

```diff
@@ -13,7 +13,7 @@
 //
 // This requires Capslock to be installed:
 //
-//	go install github.com/google/capslock/cmd/capslock@latest
+//	go install github.com/google/capslock/cmd/capslock-git-diff@latest
```

Comment:
The comment is actually correct as written, as the line above mentions Capslock needs to be installed. You can see this further down where we call the capslock binary.

https://github.com/google/capslock/blob/a52e07fb26a976babcffae34fef615595b9ba189/cmd/capslock-git-diff/main.go#L149-L162